app/views/proposals/details/_request_details.html.haml

```diff
@@ -7,4 +7,4 @@
             Request Details
         .content-content.column#view-request-details
           .row
-            = render @current_user.client_slug + '/request_details'
+            = render client_specific_partial(@proposal.requester, '/request_details')
```

Comment:
@yozlet Im not using it anymore because it has a weird partial structure. Most of the partials above are either passing `data` or have a partial prefix. The helper doesnt account for those things.